docs/config.md

```diff
@@ -4,11 +4,10 @@
 
 | Environment variable | Description | Default |
```

Comment:
```suggestion
| Environment variable | Description | Default value |
```

Reply:
For consistency, please change all the "Default" and "Value" columns to "Default value" (if that is indeed their meaning). For example, see the tables from line 127 onwards.

Reply:
Done in 6d5921ac1ad1b0e9602f1055f5fa527f1e8711ec 👍🏼